software/externals/swigmake

```diff
@@ -0,0 +1 @@
+Subproject commit ab03741a0627e99589ecbc1c088a4db05755e3c2
```

Comment:
your commit adds swigmake into cmake/externals.cmake, so it will be downloaded automatically, it doesn't (shouldn't) need to be added as a git submodule.